src/meta/chart/pieChart.ts

```diff
@@ -0,0 +1,17 @@
+import type { PieLabel } from 'recharts/types/polar/Pie'
+
+import { ChartColor } from 'meta/chart/chart'
+
+export type PieChartData = {
+  name: string
+  value: number
+  color: string
+}
+
+type PieCell = { key: string; color: ChartColor }
```

Comment:
Maybe more meaningful to change to

```suggestion

type PieCell = { name: string, color: ChartColor }
export type PieChartData = PieCell & { value: string }

```